ingestion/src/metadata/ingestion/source/messaging/kinesis/models.py

```diff
@@ -0,0 +1,125 @@
+#  Copyright 2023 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+Kinesis Models
+"""
+# Disable pylint to conform to Kinesis API returns
+# We want to convert to the pydantic models in 1 go
+# pylint: disable=invalid-name
```

Comment:
flagging the pylint disable in the module where the issue originates. This might be a wrong flag from pylint on the checkstyle. However, this solves the issue